chess/authenticaion.py

```diff
@@ -0,0 +1,16 @@
+import functools
+from django.shortcuts import redirect
+from django.contrib import messages
+
+def user_required(view_func, redirect_url="index"):
```

Comment:
as it currently stands, this authentication decorator checks for the django user model and not our player model. Our next task will be to fix this Decorator to properly authenticate based on the current sign in player, and in the future add in admin pages to view our players from an admin perspective.